lib/model/edit_button.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:note_app/model/note_model.dart';
+import '../screens/edit_note_screen.dart';
+
+class EditButton extends StatefulWidget {
```

Comment:
Why using StatefulWidget over StatelessWidget

Reply:
because we need a constructor
to define 'note'

Reply:
because we need a constructor
to define 'note'